app/models/address.rb

```diff
@@ -30,18 +31,46 @@ def full_address
 
   alias to_s full_address
 
+  def geocode_with_ward
+    # TODO? Is there any point using the Geocoder gem rather than writing
+    # our own code to query postcodes.io given that using postcodes.io
+    # now defines how we use Geocoder?
+    # - Passing anything other than a postcode will cause Geocoder.search to
+    #   return an empty array.
+    # - We are accessing admin_ward directly through through
+    #   Geocoder::Result::PostcodesIo#data hash.
+
+    geo = Geocoder.search(postcode).first&.data
+
+    if geo
```

Comment:
`return unless geo`

Reply:
Your suggestion fits well with the coding style I've seen in the app so far. I prefer to minimise method exit points but that's probably old skool now. I'll do what you said

Reply:
Is that tool widely used by the community? Although I don't like the suggested style in this case, I can see the benefit of the community tending towards a single style.

Reply:
Tends to be, you can config it how you like, but there is a sound justification for most of these things. I think in this case it's that indentention just reduces readability, esp in increasingly nested blocks. maybe something like `return false unless geo` is clearer?

Reply:
Aside:
The linux people used to mandate that coders set indents to 8 spaces. They said that if you had more than 2 levels of nesting then your code was unreadable anyway so having all slide off the side of the screen was good wake up call :laughing: